evm_arithmetization/src/verifier.rs

```diff
@@ -249,12 +250,16 @@ pub mod testing {
 
         // Extra sums to add to the looked last value.
         // Only necessary for the Memory values.
-        let mut extra_looking_sums = vec![vec![F::ZERO; config.num_challenges]; NUM_TABLES];
+        let mut extra_looking_sums =
```

Comment:
I think we can use `let mut extra_looking_sums = HashMap::new()`

Reply:
This causes the proving jobs to fail (two artifacts blocks) if I don't initialize each individual sum with `vec![F::ZERO; config.num_challenges]`

Reply:
Just curious, the unit tests passed, so why did the proving blocks job fail?

Reply:
There is only 1 recursive unit test (apart from consistency of constraints) that is ran in CI, it's `empty_tables` that is not complete (my comment on one of your PRs actually) as it only generates segment proofs.
Initializing the `HashMap` without specifying routable wires causes the recursive verification (not native one) to fail, which is done at the layer above (segment_aggregation). You can test it on `two_to_one` test for instance (though it is ignored in the CI).

Reply:
Do you know why it fails at the layer above?